roles/wordpress-setup/tasks/nginx-client-cert.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Download client cert
+  get_url:
+    url: "{{ item.value.ssl.client_cert_url }}"
+    dest: "{{ nginx_ssl_path }}/{{ item.value.ssl.client_cert_url | hash('md5') }}"
```

Comment:
I think a user who happens upon the server's `nginx_ssl_path` contents would find the filename more intuitive as `client-4c1d4cc.crt` instead of `4c1d4ccad291c1ac1f230ad263c4196c`.

Could we make this change?
```diff
- dest: "{{ nginx_ssl_path }}/{{ item.value.ssl.client_cert_url | hash('md5') }}"
+ dest: "{{ nginx_ssl_path }}/client-{{ (item.value.ssl.client_cert_url | hash('md5'))[:7] }}.crt"
```

A similar filename change would be needed in `wordpress-site.conf.j2`.